app/ui/pegbaralignmentdialog.ui

```diff
@@ -50,21 +40,21 @@
      <item>
       <widget class="QLabel" name="labSelectCenterPegs">
        <property name="text">
-        <string>1) Select a reference area around center pegs.</string>
+        <string>1) A selection should exist</string>
        </property>
       </widget>
      </item>
      <item>
-      <widget class="QLabel" name="labSelectRefKeyframe">
+      <widget class="QLabel" name="label">
        <property name="text">
-        <string>2) Select one reference keyframe from timeline.</string>
+        <string>2) The selection should contain all pegs</string>
```

Comment:
Ah indeed, I forgot that there are usually 3 pegs... and we're referring the center peg (well that doesn't matter really, since we're just checking pixels) but the comment is still relevant. I've made it more explicit now, mentioning that it involves all frames.